image/templates/helm/stackrox-secured-cluster/templates/sensor-scc.yaml

```diff
@@ -66,6 +67,11 @@ rules:
   resources:
   - securitycontextconstraints
   resourceNames:
+{{- if ._rox.system.createSCCs }}
+  - stackrox-sensor
+{{- end }}
+  - nonroot-v2
+  - nonroot
```

Comment:
I understand that you list both `nonroot-v2` and `nonroot` here, as the former might not always be available. But what about `nonroot` and `anyuid`? Would we ever want `anyuid` to be used? Can there be cases where only `anyuid` is available, but not `nonroot`? We should be specific here as to what SCC we want to be used.

Reply:
I think I also don't quite understand why we remove the `use` verb from the `edit` role above, but then create an RBAC binding "officially" using the SCC which was previously accidentally used

Reply:
I wanted to split the work of getting rid of `anyuid` for sensor into 2 PRs:
- this one, where we refactor things to prepare everything and catch possible errors,
- and a follow-up where we remove a single line `- anyiud` and so make the sensor to use `nonroot` or `nonroot-v2`.

If we stayed with the full-powered `edit` role, the follow-up PR would not work, because the `edit` role would still be allowing to use the `anyuid` SCC.